example-workflows/gke/README.md

```diff
@@ -1,7 +1,7 @@
 # Google Kubernetes Engine - GitHub Actions
 
-An example workflow that uses [GitHub Actions][actions] to deploy [a static
-website](site/) to an existing [Google Kubernetes Engine][gke] cluster.
+An example workflow that uses [GitHub Actions][actions] to deploy a simple nginx server
```

Comment:
Would something like us-docker.pkg.dev/google-samples/containers/gke/hello-app:1.0 be better as a hello world image?

Reply:
Finally decided to use this: https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/tree/master/hello-app